src/diffusers/utils/testing_utils.py

```diff
@@ -132,6 +132,33 @@ def numpy_cosine_similarity_distance(a, b):
     return distance
 
 
+def check_if_dicts_are_equal(dict1, dict2):
+    for key, value in dict1.items():
+        if isinstance(value, set):
+            dict1[key] = sorted(value)
+    for key, value in dict2.items():
+        if isinstance(value, set):
+            dict2[key] = sorted(value)
+
+    for key in dict1:
+        if key not in dict2:
+            raise ValueError(
```

Comment:
I would either return `False` here and let the caller assert that `check_if_dicts_are_equal` is `True`, or raise an `AssertionError` and not return `True` at the end.